app/db_services/user_service.py

```diff
@@ -57,17 +57,15 @@ def get_user_by_id(user_id):
     else:
         return None
 
-def update_user(user_id, email):
+def update_user(user_id, mobileno):
     try:
-        updated_data = {"$set": {"email": email}}
+        updated_data = {"$set": {"mobileno": mobileno}}
```

Comment:
I can update any user related data, except email, So, It's not like you hardcode each and every field